charactersheet/charactersheet/viewmodels/character/status_line.js

```diff
@@ -34,7 +34,11 @@ function StatusLineViewModel() {
         if (!profile || statuses.length == 0) { return ''; }
 
         return profile.characterName() + ' is ' + statuses.map(function(e, i, _) {
-            return '<span class="text-' + e.type() + '">' + e.name() + '</span>';
-        }).join('&nbsp;,') + '.';
+            if (statuses.length > 1 && i == statuses.length - 1) {
+                return 'and <span class="text-' + e.type() + '">' + e.name() + '</span>';
```

Comment:
`something, something, and something` is an example of the oxford comma. It's correct grammatically.